code/FulpstationCode/fulp_starter_gear/fulp_starter_gear.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/storage/belt/security/fulp_starter_full/PopulateContents()
+	new /obj/item/reagent_containers/spray/pepper(src)
+	new /obj/item/flashlight/seclite(src)
+	new /obj/item/radio(src)
+	new /obj/item/assembly/flash/handheld(src)
+	new /obj/item/melee/baton/loaded(src)
+	update_icon()
+
+/obj/item/storage/box/security/improved/PopulateContents()
+	..() // we want the regular stuff too
+	new /obj/item/crowbar/red(src)
+	new /obj/item/flashlight/glowstick(src)
+	new /obj/item/flashlight/glowstick(src)
+	new /obj/item/flashlight/glowstick(src)
```

Comment:
Adding the glowsticks I feel is kind of redundant since they're already spawning with a Seclight. Assuming #168 is merged as well this would give them the civilian glowsticks rather then the military ones; it makes the light pouch fairly redundant too I feel.

Reply:
I have no strong opinions on this one; I could remove the glowsticks.

Reply:
Also, during testing I learned something interesting: you can't simply null out backpack contents to properly override; you must utilize an empty list instead.